certify-service/src/main/resources/application-local.properties

```
@@ -206,7 +206,6 @@ mosip.certify.mock.vciplugin.verification-method=${mosip.certify.authn.jwk-set-u
 mosip.certify.mock.authenticator.get-identity-url=http://localhost:8082/v1/mock-identity-system/identity
 #TODO: get the secret for key-cert onboarded for local test
 mosip.certify.mock.vciplugin.issuer.key-cert="dummy-issuer-cert"
```

```suggestion
mosip.certify.mock.vciplugin.issuer.key-cert=dummy-issuer-cert
```

This might be a better way to include filename configs as it won't include double quotes. Please enhance the code accordingly to fetch the files when the server is deployed with Spring Cloud Config.